.ci.yaml

```diff
@@ -322,12 +322,14 @@ targets:
       version_file: flutter_master.version
       target_file: linux_custom_package_tests.yaml
       # Pigeon tests need Andoid deps (thus the Linux_android base), emulator,
-      # and clang-format.
+      # and Linux desktop deps.
```

Comment:
Was the comment about clang-format intentionally removed? I think it's still used in the tests right?

Reply:
Clang is part of the Linux desktop deps, so I was simplifying the comment rather than have two different explanations for why clang was in the list.